### ISSUE TITLE
Keyboard Navigation for the lightbox

### ISSUE THREAD

**Issue by uncovery** [uncovery/unc_gallery#26]
The lightbox is the wordpress-internal thickbox. There should be a keyboard navigation and the arrows should be on the edges instead of the description

**Comment by uncovery** [comment]
This is the wordpress codex page: https://codex.wordpress.org/Javascript_Reference/ThickBox
We might have to use a different, external thickbox instead if we want more functionality.

**Comment by uncovery** [comment]
Will switch to http://lokeshdhakar.com/projects/lightbox2/

**Comment by uncovery** [comment]
done.
